src/actions/console-actions.js

```diff
@@ -0,0 +1,23 @@
+import evalQueue from "./evaluation-queue";
+
+export function evalConsoleInput(consoleText) {
```

Comment:
moved this logic over from the eval frame; this is now triggered by a message from the eval frame